persistence/src/main/java/cz/fi/muni/pa165/entity/Visit.java

```diff
@@ -101,4 +101,16 @@ public int hashCode() {
         result = 31 * result + (date != null ? date.hashCode() : 0);
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "Visit{" +
+                "id=" + id +
+                ", hunter=" + hunter +
```

Comment:
this will call toString on hunter, forest, etc. So the output will be quite superfluous. I guess it is a matter of preference.